alloydb/notebooks/embeddings_batch_processing_e2e_test.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2022 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This sample creates a secure two-service application running on Cloud Run.
+# This test builds and deploys the two secure services
+# to test that they interact properly together.
+
+import asyncpg  # type: ignore
+import conftest as conftest  # python-docs-samples/alloydb/conftest.py
+from google.cloud.alloydb.connector import AsyncConnector, IPTypes
+import pytest
+import sqlalchemy
+from sqlalchemy.ext.asyncio import AsyncEngine, create_async_engine
+
+
+def preprocess(source: str) -> str:
+    # Skip the cells which add data to table
+    if "df" in source:
+        return ""
+    # Skip the colab auth cell
+    if "colab" in source:
+        return ""
+    return source
+
+
+async def _init_connection_pool(
+    connector: AsyncConnector,
+    db_name: str,
+    project_id: str,
+    cluster_name: str,
+    instance_name: str,
+    region: str,
+    password: str,
+) -> AsyncEngine:
+    connection_string = (
+        f"projects/{project_id}/locations/"
+        f"{region}/clusters/{cluster_name}/"
+        f"instances/{instance_name}"
+    )
+
+    async def getconn() -> asyncpg.Connection:
+        conn: asyncpg.Connection = await connector.connect(
+            connection_string,
+            "asyncpg",
+            user="postgres",
+            password=password,
+            db=db_name,
+            ip_type=IPTypes.PUBLIC,
+        )
+        return conn
+
+    pool = create_async_engine(
+        "postgresql+asyncpg://",
+        async_creator=getconn,
+        max_overflow=0,
+    )
+    return pool
+
+
+@pytest.mark.asyncio
+async def test_embeddings_batch_processing(
+    project_id: str,
+    cluster_name: str,
+    instance_name: str,
+    region: str,
+    database_name: str,
+    password: str,
+    table_name: str,
+) -> None:
+    # TODO: Create new table
+    # Populate the table with embeddings by running the notebook
+    conftest.run_notebook(
+        "alloydb/notebooks/embeddings_batch_processing.ipynb",
```

Comment:
This path needs to be relative
```suggestion
        "embeddings_batch_processing.ipynb",
```